docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -581,7 +581,7 @@
   #   pvc: &lt;name&gt;                           # The name of the persistant volume claim
   #   forcePermissions: false
   #   hosts: [ &lt;host-list&gt; ]                 # Inline list of hosts
-  #   hostGroup: &lt;name&gt;                     # Name of the host group for bare metal 
+  #   hostGroup: &lt;name&gt;                     # Name of the host group for bare-metal 
```

Comment:
in this one case only displays the content of the yaml file so this change may also need to get added to the default yaml file as well?